### PR TITLE
feat: add initial interfaces [SC-6425]

### PR DESCRIPTION
**Pull request by natrad100**

### CODE REVIEW

File: maritime_interfaces/sensors/battery/Battery.msg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+std_msgs/Header header
```
Comment (russkel) [inline maritime_interfaces/sensors/battery/Battery.msg]: Yeah this doesn't have any value. Should get rid of this message.

File: maritime_interfaces/sensors/generic/Pressure.msg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+std_msgs/Header header
```
Comment (russkel) [inline maritime_interfaces/sensors/generic/Pressure.msg]: Get rid of all thisc message.

File: maritime_interfaces/engineering/engine/EngineState.msg [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# EngineState.msg
+
+std_msgs/Header header
+
+float32 speed
+float32 throttle
+maritime_interfaces_support/GenericSensorData turbo_state
+maritime_interfaces_support/GenericSensorData intake_state
```
Comment (russkel) [inline maritime_interfaces/engineering/engine/EngineState.msg]: Don't have generic sensor data. This is not ROS2 convention.

File: maritime_interfaces/support/shared/FluidData.msg [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# FluidData.msg
```
Comment (russkel) [inline maritime_interfaces/support/shared/FluidData.msg:1]: Flatten this message.

File: maritime_interfaces/sensors/generic/Temperature.msg [hunk 5]
```diff
@@ -0,0 +1,3 @@
+std_msgs/Header header
```
Comment (russkel) [inline maritime_interfaces/sensors/generic/Temperature.msg]: Get rid of this message.

File: maritime_interfaces/support/shared/GenericSensorData.msg [hunk 6]
```diff
@@ -0,0 +1,4 @@
+std_msgs/Header header
```
Comment (russkel) [inline maritime_interfaces/support/shared/GenericSensorData.msg]: Remove this message.

File: maritime_interfaces/support/shared/ComponentInfo.msg [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# ComponentInfo.msg
```
Comment (russkel) [inline maritime_interfaces/support/shared/ComponentInfo.msg:1]: Does this exist as a CAN message already?

Comment (natrad100) [reply maritime_interfaces/support/shared/ComponentInfo.msg:1]: It does, but doesn't mean we don't want something on the other side of the integration service.

**Pull request merged**